source/_integrations/sensor.rest.markdown

```diff
@@ -115,6 +115,10 @@ resource_template:
   description: The resource or endpoint that contains the value with template support.
   required: true
   type: template
+scan_interval:
+  description: Define the refrequency to call the REST endpoint in seconds.
+  required: false
+  type: integer
```

Comment:
Scan interval is not an option for this platform integration but is provided by home assistant.

It is documented here: <https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>